Documentation/media-migration.md

```diff
@@ -1,220 +1,828 @@
-# Migrate from Media Analytics SDK to AEP SDK on Roku platform
+# Migrate from Media SDK to AEP SDK
+
+## Table of contents
+| Sections |
+| -- |
+| [Prerequisistes](#prerequisistes) |
+| [API comparison between SDKs](#api-comparison) |
+| [Initialize SDK instance](#initialize-sdk-instance) |
+| [Start Media session](#start-media-session) |
+| [Track Media events](#track-media-events) |
+
+## Prerequisistes
+1. [Experience Data Model (XDM)](https://experienceleague.adobe.com/docs/experience-platform/xdm/home.html?lang=en)
+2. [Datastreams](https://developer.adobe.com/client-sdks/home/getting-started/configure-datastreams/)
+3. [Getting Started with implementing Media tracking using AEP Roku SDK](./getting-started.md)
+4. [Setup schemas, datastream, dataset, Customer Journey Analytics (CJA) dashboard etc](https://experienceleague.adobe.com/docs/media-analytics/using/implementation/edge-recommended/media-edge-sdk/implementation-edge.html)
+
+
+## API comparison
+
+> [!NOTE]
+> AEP SDK has only two media APIs `createMediaSession()` and `sendMediaEvent()`.
+
+### Core Plaback APIs:
+
+| Media SDK | AEP SDK|
+| -- | -- |
+| `mediaTrackSessionStart(mediaInfo,mediaContextData)` | `createMediaSession(sessionStartXDM)` |
+| `mediaTrackPlay()` | `sendMediaEvent(playXDM)` |
+| `mediaTrackPause()` | `sendMediaEvent(pauseStartXDM)` |
+| `mediaTrackComplete()` | `sendMediaEvent(sessionCompleteXDM)` |
+| `mediaTrackSessionEnd()` | `sendMediaEvent(sessionEndXDM)` |
+
+### Ad Tracking APIs:
+| Media SDK | AEP SDK|
+| -- | -- |
+| `mediaTrackEvent(ADBMobile().MEDIA_AD_BREAK_START, adBreakInfo, contextData)` | `sendMediaEvent(adbreakStartXDM)` |
+| `mediaTrackEvent(ADBMobile().MEDIA_AD_BREAK_COMPLETE, invalid, invalid)` | `sendMediaEvent(adbreakCompleteXDM)` |
+| `mediaTrackEvent(ADBMobile().MEDIA_AD_START, adInfo, contextData)` | `sendMediaEvent(adStartXDM)` |
+| `mediaTrackEvent(ADBMobile().MEDIA_AD_COMPLETE, invalid, invalid)` | `sendMediaEvent(adCompleteXDM)` |
+| `mediaTrackEvent(ADBMobile().MEDIA_AD_SKIP, invalid, invalid)` | `sendMediaEvent(adSkipXDM)` |
+
+### Buffer and Seek APIs
+| Media SDK | AEP SDK|
+| -- | -- |
+| `mediaTrackEvent(ADBMobile().MEDIA_BUFFER_START, invalid, invalid)` | `sendMediaEvent(bufferStartXDM)` |
+| `mediaTrackEvent(ADBMobile().MEDIA_BUFFER_COMPLETE, invalid, invalid)` | `sendMediaEvent(bufferCompleteXDM)` |
+| `mediaTrackEvent(ADBMobile().MEDIA_SEEK_START, invalid, invalid)` | `sendMediaEvent(pauseStartXDM)` |
+| `ADBMobile().mediaTrackEvent(ADBMobile().MEDIA_SEEK_COMPLETE, invalid, invalid)` | `sendMediaEvent(pauseStartXDM)` |
+
+> [!NOTE]
```

Comment:
Can you move this note below the heading? Also reword 
`Media backend will detect seek based on change in playhead and timestamp values.`